_posts/2015-11-25-mouseless-programming.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+name: mouseless-programming
+title: 'Mouseless Programming'
+date: 2015-11-25 17:00:00 +00:00
+author: Tomaz Tekavec
+image:
+    src: "/assets/img/custom/blog/dilbert.jpg"
+    attribution:
+        text: Scott Adams, Still Pumped from Using the Mouse SC 
+        href: https://d1466nnw0ex81e.cloudfront.net/n_iv/600/1038055.jpg
+tags:
+- mouseless
+- programming
+- productivity
+- IDE
+---
+
+## Mouseless Programming
+
+To be honest, I used to be quite a power mouse user. Looking back and trying to figure out why I got infected, it probably happened when Microsoft released Word 6.0 with its twenty-some toolbars with thirty-some buttons in each toolbar. I didn't remember many shortcuts, except ctrl+c, ctrl+v, and ctrl+s, which I did learn and instantly became a full-stack Word guru in the eyes of toolbar-only users.
+
+Slowly, but inevitably, IntelliJ IDEA and Visual Studio replaced my command-line Turbo Pascal and Fortran compilers, few years later one good guy showed me ReSharper... Still, I'd rather made jokes about 3,721 IDE shortcuts and Spiderman skills to execute 9-fingers keyboard combinations instead of learning one shortcut per week � and hence my right hand became a devoted Logitech docking station.
+
+But the truth is, when we're coding, by each move from the keyboard to the mouse and back, we lose some time. It might not be a big number in terms of seconds, but during the day it's not a negligible sum. And there's another side effect � if you need mouse often, it's tempting to have several toolbars and nice gadget frames visible, which in turn decreases the available space for our core business, coding. 
+To get rid of this bad habit, I recorded two cases, a comparison between performing some common coding tasks using a mouse, and without it. In the videos below, you can see the result.
+
+#### Part I - Programming With Mouse and Keyboard
+
+<iframe width="694" height="390" src="https://www.youtube.com/embed/g6kfdlUZARs" frameborder="0" allowfullscreen></iframe>
+
+#### Part II - Programming With Keyboard Only
+
+<iframe width="694" height="390" src="https://www.youtube.com/embed/Rcf6cwP_J8M" frameborder="0" allowfullscreen></iframe>
+
+
```

Comment:
Maybe put the results of the experiment here (5minutes vs 6minutes) for the impatients ?